src/lib/Scenes/Map/GlobalMap.tsx

```diff
@@ -535,15 +535,17 @@ export class GlobalMap extends React.Component<Props, State> {
                   })
                 }}
               />
-              <Box style={{ marginLeft: "auto" }}>
-                <UserPositionButton
-                  highlight={this.state.userLocation === this.state.currentLocation}
-                  onPress={() => {
-                    const { lat, lng } = this.state.userLocation
-                    this.map.moveTo([lng, lat], 500)
-                  }}
-                />
-              </Box>
+              {this.state.userLocation && (
```

Comment:
Maybe add a default `null` state for `userLocation`?

Reply:
It's an optional in `State`, I think `undefined` makes sense as a default?